src/charts/donut/DonutComponent.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import donut from './donutChart';
+
+
+export default class Donut extends Component {
+
+    static PropTypes = {
+        data: PropTypes.array.isRequired,
+
+        colorSchema: PropTypes.arrayOf(PropTypes.string),
+        externalRadius: PropTypes.number,
+        hasFixedHighlightedSlice: PropTypes.boolean,
+        height: PropTypes.number,
+        highlightSliceById: PropTypes.number,
+        internalRadius: PropTypes.number,
+        isAnimated: PropTypes.boolean,
+        margin: PropTypes.shape({
+            top: PropTypes.number,
+            bottom: PropTypes.number,
+            left: PropTypes.number,
+            right: PropTypes.number,
+        }),
+        orderingFunction: PropTypes.func,
+        width: PropTypes.number,
+
+        customMouseOver: PropTypes.func,
+        customMouseMove: PropTypes.func,
```

Comment:
I think this one won't be necessary for the donut chart

Reply:
Let's remove it then :)